testing/test_main.py

```diff
@@ -1,5 +1,8 @@
+from typing import Optional
+
 import pytest
-from _pytest.config import ExitCode
```

Comment:
Should choose this one not the `main` one.

Reply:
Oh good catch, fixed! 👍